src/styles/product-card.scss

```diff
@@ -0,0 +1,111 @@
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+:root {
+  --color-for-main-text: #060B35;
+  --color-for-secondary-text: #616070;
+  --color-for-button: #00ACDC;
+  --color-for-background: #fff;
+  --color-for-button-text: #fff;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
These are general classes so this part should be in a separate file it's not related to the  `product-card`